src/tracer.js

```diff
@@ -96,7 +100,18 @@ export default class Tracer {
     this.registerInjector(opentracing.FORMAT_BINARY, binaryCodec);
     this.registerExtractor(opentracing.FORMAT_BINARY, binaryCodec);
 
-    this._reporter.setProcess(this._serviceName, Utils.convertObjectToTags(this._tags));
+    const uuid = Utils.getRandom64().toString('hex');
```

Comment:
`uuid` is a misnomer here and in remote throttler as the value is not actually a uuid.